hphp/tools/gdb/gdbutils.py

```diff
@@ -300,3 +304,11 @@ def deref(val):
         return val.cast(rawtype(val.type))
     else:
         return deref(p.referenced_value())
+
+def get_arch(self):
```

Comment:
This function needs to be documented with a triple-quote docstring, as all the functions above.